packages/calcite-components/src/components/pagination/pagination.tsx

```diff
@@ -112,12 +112,16 @@ export class Pagination
   /** Specifies the total number of items. */
   @Prop({ reflect: true }) totalItems = 0;
 
-  @Watch("pageSize")
   @Watch("totalItems")
-  handleTotalPages(): void {
+  handleTotalItems(): void {
     this.totalPages = this.totalItems / this.pageSize;
```

Comment:
It seems like `handleTotalPages` and `handleTotalItems` does the same logic, anyway to combine them?